grepos/app/src/main/java/net/boondockradio/grepos/MainActivity.java

```diff
@@ -22,51 +23,78 @@
 
 public class MainActivity extends AppCompatActivity {
 
-    private final String TAG = "grepos";
+    private final String TAG = MainActivity.class.getSimpleName();
+    private final int LOAD_MORE_THRESHOLD = 1;
+    private final String PER_PAGE = "50";
 
     private List<Repository> mRepositoryItems;
```

Comment:
you can declare it here, i think its ok... I do it all the time

`private List<Repository> mRepositoryItems = new ArrayList();`

For me its safer to do it here. mRepositoryItems is never null.